src/dotless.Core/Parser/Tree/Ruleset.cs

```diff
@@ -279,6 +281,50 @@ protected void EvaluateRules(Env env)
             env.Frames.Pop();
         }
 
+        /// <summary>
+        /// Merges properties with merge operators '+' and '+_'
+        /// </summary>
+        /// <returns>New merged ruleset</returns>
+        private NodeList MergeProperties()
+        {
+            var groups = new Dictionary<string, List<Rule>>();
+            var sorted = new List<Node>();
+            foreach (var node in Rules)
+            {
+                var rule = node as Rule;
+                if (rule == null || rule.Merge == "")
+                {
+                    sorted.Add(node);
+                    continue;
+                }
+                var key = $"{rule.Name},{rule.Important},{rule.Merge}";
+                if (!groups.ContainsKey(key))
+                {
+                    groups[key] = new List<Rule> {rule};
+                    sorted.Add(node);
+                }
+                else
+                {
+                    if (rule.Merge == "")
+                        sorted.Add(rule);
+                    else
+                        groups[key].Add(rule);
+                }
+            }
+            foreach (var group in groups.Values)
+            {
+                var rule = @group[0];
+                var value = rule.Value as Value;
+                if (value == null)
+                    value = (Value) (rule.Value = new Value(new[] {rule.Value}, "", rule.Merge));
```

Comment:
The case is when value is a number and not a value node.

Numbers contain only one value but the value object holds a list of values, for example border can have 3 values, width, style and color.

But I will refactor to better indicate what is going on.

The value object already handles the merging of the different separate values so I just rebuild the separate rules into one rule with multiple values and let the existing value class do the actual concatenation instead of creating a new almost identical class.

But  padding+_: 10px; will create a rule with a single numeric value that cannot take any extra so I have to replace the num value with a values class value containing the original num value as first element.